tests/model_registry/rbac/test_mr_rbac.py

```diff
@@ -53,144 +60,71 @@
-
-        Args:
-            model_registry_instance: The Model Registry instance to test against
-            model_registry_namespace: The namespace where Model Registry is deployed
-            admin_client: The admin client for accessing the cluster
-            test_idp_user_session: The test user session containing both admin and user contexts
-            use_admin_context: Whether to use admin context (True) or user context (False)
-
-        Raises:
-            AssertionError: If access permissions don't match expectations
-            ForbiddenException: Expected for normal users, unexpected for admin users
+        This test verifies that admin users can access the Model Registry
         """
-
-        context_to_use = (
-            test_idp_user_session.original_context if use_admin_context else test_idp_user_session.user_context
+        client_args = build_mr_client_args(
+            token=get_openshift_token(), rest_endpoint=model_registry_instance_rest_endpoint
         )
+        mr_client = ModelRegistryClient(**client_args)
+        assert mr_client is not None, "Client initialization failed for admin user"
+        LOGGER.info("Successfully created Model Registry client for admin user")
 
-        with switch_user_context(context_to_use):
-            _, client_args = get_mr_client_args(
-                model_registry_instance=model_registry_instance,
-                model_registry_namespace=model_registry_namespace,
-                admin_client=admin_client,
-            )
-
-            if use_admin_context:
-                mr_client = ModelRegistryClient(**client_args)
-                assert mr_client is not None, "Client initialization failed for admin user"
-                LOGGER.info("Successfully created Model Registry client for admin user")
-            else:
-                with pytest.raises(ForbiddenException) as exc_info:
-                    _ = ModelRegistryClient(**client_args)
-                assert exc_info.value.status == 403, f"Expected HTTP 403 Forbidden, but got {exc_info.value.status}"
-                LOGGER.info("Successfully received expected HTTP 403 status code")
+    @pytest.mark.sanity
+    def test_user_permission_non_admin_user(
+        self: Self,
+        test_idp_user_session: Generator[UserTestSession, None, None],
+        model_registry_instance_rest_endpoint: str,
+        login_as_test_user: None,
+    ):
+        """
+        This test verifies that non-admin users cannot access the Model Registry (403 Forbidden)
+        """
+        client_args = build_mr_client_args(
+            rest_endpoint=model_registry_instance_rest_endpoint, token=get_openshift_token()
+        )
+        with pytest.raises(ForbiddenException) as exc_info:
+            ModelRegistryClient(**client_args)
+        assert exc_info.value.status == 403, f"Expected HTTP 403 Forbidden, but got {exc_info.value.status}"
+        LOGGER.info("Successfully received expected HTTP 403 status code")
```

Comment:
Are you sure `.value.status` exists? I think the only supported syntax is `.value` from pytest: https://docs.pytest.org/en/stable/reference/reference.html#pytest.ExceptionInfo

Reply:
I did test with changing this value and it worked. Also, this code was not changed https://github.com/opendatahub-io/opendatahub-tests/blob/main/tests/model_registry/rbac/test_mr_rbac.py#L119. So far I see it is working fine.